### ISSUE TITLE
Stream entryid null will cause error

### ISSUE THREAD

**Issue by wmlamaa** [oliver006/redis_exporter#823]
**Describe the problem**
if the stream is newly created, its first entry and last entry will be null in redis and the exporter seems expected the entry id always has value, so it is causing an error in this case.

**What version of redis_exporter are you running?**
Please run `redis_exporter --version` if you're not sure what version you're running.
[ ] 0.3x.x
[X] 1.x.x

**Running the exporter**
What's the full command you're using to run the exporter? (please remove passwords and other sensitive data)
run in docker with the args : --debug=true --export-client-list=true --include-system-metrics=true 

**Expected behavior**
What metrics are missing? What metrics are wrong? Is something missing that was present in an earlier version? 
Did you upgrade from 0.3x.x to 1.0 and are scraping multiple hosts? [Have a look here ](https://github.com/oliver006/redis_exporter#prometheus-configuration-to-scrape-multiple-redis-hosts) how the configuration changed.

**Screenshots**

**Additional context**

**Comment by oliver006** [comment]
Thanks for raising this issue.
Does the exporter crash or just output an error? Do you have a screenshot or logs from the error?

**Comment by wmlamaa** [comment]
The app will crash. Here is the logs for the error. 

```
panic: interface conversion: interface () is nil, not []interface ()

goroutine 26 [running]:
github.com/oliver006/redis_exporter/exporter.getEntryId(...)
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/src/github.com/oliver006/redis_exporter/exporter/streams.go:70
github.com/oliver006/redis_exporter/exporter.getstreamInfo((0x9f8e20, 0xc0001270e0), [0xc000339240, 0x1b])
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/src/github.com/oliver006/redis_exporter/exporter/streams.go:55 +0x411
github.com/oliver006/redis_exporter/exporter.(*Exporter).extractStreamMetrics(0xc00012c240, 0x5?, (0x9f8e20, 0xc0001270e0])
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/src/github.com/oliver006/redis_exporter/exporter/streams.go:178 +0x554
github.com/oliver006/redis_exporter/exporter.(*Exporter).scrapeRedisHost(0xc00012c240, Oxc0001003c0%)
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/src/github.com/oliver006/redis_exporter/exporter/exporter.go:645 +0xe9b
github.com/oliver006/redis_exporter/exporter.(*Exporter).Collect(0xc00012c240, 0xc00004bf60?)
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/src/github.com/oliver006/redis_exporter/exporter/exporter.go:471 +0xef
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/pkg/mod/github.com/prometheus/client_golang@v1.16.0/prometheus/registry.go:455 +0x16
eated by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
/bitnami/blacksmith-sandox/redis-exporter-1.52.0/pkg/mod/github.com/prometheus/client_golang@v1.16.0/prometheus/registry.go:547
```

**Comment by oliver006** [comment]
That's not good - let me have a look.

**Comment by oliver006** [comment]
`if the stream is newly created, its first entry and last entry will be null in redis` - how did you do that? I don't have much experience with streams so not sure how I can create an empty stream.

**Comment by wmlamaa** [comment]
sth like this `xgroup create newstream newgroup $ mkstream`. Then you will have a empty stream with null first-entry and last-entry

**Comment by oliver006** [comment]
Thanks, that helped with reproducing it. I just open #824 which should hopefully fix this.

**Comment by wmlamaa** [comment]
thanks @oliver006 . Appreciate your help.

**Comment by oliver006** [comment]
Fix looks good but I want to add a test to this before it gets merged. I'll probably get this wrapped tomorrow or in the next few days.

**Comment by oliver006** [comment]
Fixed in v1.53.0
